apps/settings/locales/settings.en-US.properties

```diff
@@ -325,11 +325,11 @@ cellularAndData=Cellular & Data
 # If you need to adapt your localization to the reduced space available
 # in headers, you can use a shorter string here.
 cellularAndData-header={{cellularAndData}}
-dataNetwork=Carrier
+dataNetwork=Operator
```

Comment:
for strings we have to rename the identifer `dataNetwork` to other, so the tool can scan the difference and provide the changed strings to l10n team. So I suggest you to change `dataNetwork` to `dataOperator` and update the related html file.